bsdlib/doc/images/bsd_lib_memory.svg

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<!-- Generated by Microsoft Visio, SVG Export bsd_lib_memory.svg Page-1 -->
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:ev="http://www.w3.org/2001/xml-events"
+		width="2.97855in" height="4.02499in" viewBox="0 0 214.456 289.799" xml:space="preserve" color-interpolation-filters="sRGB"
+		class="st9">
+	<style type="text/css">
+	<![CDATA[
+		.st1 {fill:#00a9ce;stroke:#000000;stroke-linecap:butt;stroke-opacity:0.53;stroke-width:0.529963}
+		.st2 {fill:#0033a0;stroke:#000000;stroke-linecap:butt;stroke-opacity:0.53;stroke-width:0.529963}
+		.st3 {fill:none;stroke:none;stroke-linecap:butt;stroke-width:0.75}
+		.st4 {fill:#ffffff;font-family:Calibri;font-size:0.75em}
+		.st5 {font-size:1em}
+		.st6 {fill:#00a9ce;stroke:#000000;stroke-linecap:butt;stroke-opacity:0.53;stroke-width:0.751858}
+		.st7 {fill:#000000;font-family:Arial;font-size:0.75em}
+		.st8 {fill:#000000;font-family:Calibri;font-size:0.75em}
+		.st9 {fill:none;fill-rule:evenodd;font-size:12px;overflow:visible;stroke-linecap:square;stroke-miterlimit:3}
+	]]>
+	</style>
+
+	<g>
+		<title>Page-1</title>
```

Comment:
Are titles in SVG images used?

Reply:
I've no idea why github is even showing a text output of an svg file...this title is not visible anywhere.